tools/mOTUs/motus_merge.xml

```diff
@@ -0,0 +1,90 @@
+    <command detect_errors="exit_code"><![CDATA[
+    motus merge
+    #if $input_options.input_source == "files"
+        -i '${','.join([str($file) for $file in $input_options.files])}'
+    #elif $input_options.input_source == "directory"
+        -d '$input_options.directory'
+    #end if
+        #if $add_profiles
+        -a '$add_profiles'
+    #end if
+    -o '$output'
+    #if $biom_format
+        -B
+    #end if
+    ]]></command>
+    <inputs>
+        <conditional name="input_options">
+            <param name="input_source" type="select" label="Input source">
+                <option value="files">Input files</option>
+                <option value="directory">Input directory</option>
+            </param>
+            <when value="files">
+                <param name="files" type="data" format="txt" multiple="true" label="mOTU profiles to merge" />
+            </when>
+            <when value="directory">
+                <param name="directory" type="text" label="Directory containing mOTU profiles" help="Specify the directory path." />
+            </when>
+        </conditional>
+        <param name="add_profiles" type="text" label="Add pre-computed profiles" help="Add pre-computed profiles from different environmental samples" optional="true"/>
+        <param name="biom_format" type="boolean" truevalue="-B" falsevalue="" checked="false" label="Output in BIOM format?" />
+    </inputs>
+    <outputs>
+        <data name="output" format="txt" label="${tool.name} on ${on_string}: Merged taxon relative abundance tables"/>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="1">
+            <conditional name="input_options">
+                <param name="input_source" value="files"/>
+                <param name="files" value="SAMEA2466887-default.motus,SAMEA2466889-default.motus"/>
+            </conditional>
+            <output name="output" ftype="txt">
+                <assert_contents>
+                    <has_text text="consensus_taxonomy"/>
+                    <has_text text="SAMEA2466887"/>
+                    <has_text text="SAMEA2466889"/>
+                    <has_line_matching expression="#.*"/>
+                </assert_contents>
+            </output>
+        </test>
+        <test expect_num_outputs="1">
+            <conditional name="input_options">
+                <param name="input_source" value="files"/>
+                <param name="files" value="SAMEA2466887-c.motus,SAMEA2466889-c.motus"/>
+            </conditional>
+            <output name="output" ftype="txt">
+                <assert_contents>
+                    <has_text text="consensus_taxonomy"/>
+                    <has_text text="SAMEA2466887"/>
+                    <has_text text="SAMEA2466889"/>
+                    <has_line_matching expression="#.*"/>
+                </assert_contents>
+            </output>
+        </test>
+    </tests>
+    <help><![CDATA[
+        motus merge [options]
+
+        Input options:
```

Comment:
This is not very useful as a Galaxy help section. All those information are provided as parameters in the GUI.